src/server/pegasus_write_service.cpp

```diff
@@ -326,7 +375,7 @@ void pegasus_write_service::set_default_ttl(uint32_t ttl) { _impl->set_default_t
 
 void pegasus_write_service::clear_up_batch_states()
```

Comment:
The function name is not accurate, not only 'clean up', but also update the metrics.

Reply:
We could name this function `batch_finish()` as opposed to `batch_prepare`: finish batch write with metrics such as latencies calculated and some states cleared.